Documentation/ListModule/UsingEffectively/Index.rst

```diff
@@ -5,13 +5,13 @@
 Introduction
 ============
 
-The :guilabel:`Web > List` module allows you to browse through
-each page or folder within your site and view all of the records that
-are stored within it. The :guilabel:`List` module also gives you the ability to
-create and manage records that don't have a dedicated module.
+The :guilabel:`Web > List` module allows you to browse through pages and folders
+in your site and view the records that
+are stored there. You can also use the :guilabel:`List` module to
+create and manage records without their own module.
 
-Sub pages, content elements and news stories are examples of the different
-kinds of records that can be stored on any given page.
+Records stored on pages include sub pages, content elements and news
+stories.
```

Comment:
```suggestion
Records stored on pages include sub pages, content elements and news
stories.
```
"News stories" sounds strange here, I think usually we call them "News records". Should we mention that it depends on the setup of the system (installed use rights) and your user rights what you can actually see here?